### PR TITLE
Fixes #3: Allow upgrading a venv

### LINKED ISSUE #3: Allow upgrading a venv
Following up on https://github.com/tweaselORG/autopy/pull/2#discussion_r1205275051:

Creating a venv with version x and then later fetching it with version y won't do anything. It will to continue to run with version x. Because we're in a venv, we're using the `python` binary from that venv. And that is linked to the Python version we created the venv with.

We should have an `upgradeVenv()` version that allows upgrading a venv to a newer Python version.

### PR DESCRIPTION
**Pull request by baltpeter**
With this PR, we should be ready for the first release.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -184,12 +190,31 @@ export const getVenv = async (options: VenvOptions) => {
...
+    if (options.pythonVersion && (await fse.pathExists(venvDir)) && (await isVenv(venvDir))) {
+        const pyvenvCfg = await fse.readFile(join(venvDir, 'pyvenv.cfg'), 'utf-8');
+        const venvPythonVersion = pyvenvCfg
+            .split('\n')
+            .find((l) => l.startsWith('version = '))
+            ?.split('=')[1]
+            ?.trim();
+
+        if (venvPythonVersion && !semverSatifies(venvPythonVersion, options.pythonVersion)) {
+            if (options.allowPythonUpgrade !== false) await fse.remove(venvDir);
```
Comment (zner0L) [inline src/index.ts:203]:
```suggestion
            if (options.allowPythonUpgrade) await fse.remove(venvDir);
```

**Review by zner0L — CHANGES_REQUESTED** [review]
This is a bit simple, but fine, I guess. Thank you!

File: src/index.ts (hunk 1 above)
Comment (baltpeter) [reply src/index.ts:203]: Why wouldn't we default this to `true`?

Comment (zner0L) [reply src/index.ts:203]: Ah, now I get it!

**Pull request merged**